src/ac-power/ac-power.c

```diff
@@ -23,9 +25,9 @@ static int parse_argv(int argc, char *argv[]) {
         };
 
         static const struct option options[] = {
-                { "help",    no_argument, NULL, 'h'         },
-                { "version", no_argument, NULL, ARG_VERSION },
-                { "verbose", no_argument, NULL, 'v'         },
+                { "help" /*****/, no_argument /****/, NULL, 'h' /**********/ },
+                { "version" /**/, no_argument /****/, NULL, ARG_VERSION /**/ },
+                { "verbose" /**/, no_argument /****/, NULL, 'v' /**********/ },
```

Comment:
No, this is not an improvement. If we want to use a tool, and ugly work-arounds like that are needed, then the tool should not be enabled by default. I think it'd be reasonable to add the clang-config configuration and then commit select changes using `git add -p`.

Reply:
Or:

* accept code will be unaligned, we won't tables anymore.

Yeah, I know, aligned code is nice, but maybe the benefits of an automatic formatting tool would outweigh that.